<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: #f9f0fe;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="info_text">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>851</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser { 
	font-family: &quot;Microsoft YaHei&quot;; 
	padding: 2px;
	border-radius: 5px;
	border: 1px, solid #000000;
}
QTextBrowser:hover {
	border: 1px solid #34495E;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="send_button">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>550</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #87CEFA;
	border-radius: 10px; 
	color: rgb(0, 0, 0); 
	font-family: &quot;Microsoft YaHei&quot; 
} 
QPushButton:hover { 
	background-color: #00BFFF; 
}</string>
    </property>
    <property name="text">
     <string>Send Message!</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="input_text">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>851</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit { 
	font-family: &quot;Microsoft YaHei&quot;; 
	padding: 2px;
	border-radius: 5px;
	border: 1px, solid #000000;
}
QTextEdit:hover {
	border: 1px solid #34495E;
}</string>
    </property>
    <property name="placeholderText">
     <string>Click Basics -&gt; Connect To Server to start Chatting!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { font-family: &quot;Microsoft YaHei&quot;; }</string>
    </property>
    <property name="text">
     <string>Your Text</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>5</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { font-family: &quot;Microsoft YaHei&quot;; }</string>
    </property>
    <property name="text">
     <string>Dialog</string>
    </property>
   </widget>
   <widget class="QLabel" name="level_label">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>366</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { font-family: &quot;Microsoft YaHei&quot;; }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>892</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuServer">
    <property name="title">
     <string>Basics</string>
    </property>
    <addaction name="actionConnect_To_Server"/>
    <addaction name="actionChange_Account"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuServer"/>
  </widget>
  <action name="actionConnect_To_Server">
   <property name="text">
    <string>Connect To Server</string>
   </property>
  </action>
  <action name="actionRegister">
   <property name="text">
    <string>Register</string>
   </property>
  </action>
  <action name="actionChange_Account">
   <property name="text">
    <string>Change Account</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSend_Photo">
   <property name="text">
    <string>Send Photo</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
